src/components/pages/style.scss

```diff
@@ -38,7 +38,7 @@
 		border-bottom: 1px solid $light-gray-500;
 		color: $dark-gray-500;
 		display: flex;
-		height: 35px;
+		height: 55px;
```

Comment:
When I included the reset mixin, this height changed because `box-sizing: border-box;` was suddenly applied.

I couldn't see anything else that had changed, just this, so I've adjusted it to match the previous height.